docs/integrate/etl/influxdb.md

```diff
@@ -0,0 +1,191 @@
+(influxdb-import)=
+# Import data from InfluxDB
+
+In this quick tutorial we use [influxio](https://github.com/daq-tools/influxio)
+to import data from InfluxDB into CrateDB. It can be used both as a standalone
+program, and as a library.
+
+:::{note}
+Please note that influxio is alpha-quality software, and a work in progress.
+Contributions of all kinds are very welcome, in order to make it more solid. 
+Breaking changes should be expected until a 1.0 release, so version pinning is
+recommended, especially when you use it as a library.
+:::
+
+(setup-influxio)=
+## Setup
+
+`influxio` can be installed with pip:
+
+:::{code} console
+pip install influxio
+:::
+
+You will also need the [cratedb-toolkit](https://github.com/crate-workbench/cratedb-toolkit/tree/main/cratedb_toolkit/io#installation)
+to load the data into your CrateDB instance. It can be installed with:
+
+:::{code} console
+pip install --upgrade 'cratedb-toolkit[all]'
+:::
+
+(setup-influxdb)=
+### InfluxDB setup
+
+The following InfluxDB setup will be used in this tutorial:
+
+:::{code} console
+influx setup \
+  --username user1 \
+  --password 49jk8FQB$]1 \
+  --token token123 \
+  --org example \
+  --bucket testdrive \
+  --force
+:::
```

Comment:
That command would fail when no InfluxDB service is running.

Reply:
You can find corresponding commands to start the services using Docker on https://influxio.readthedocs.io/development.html.